ledger-tool/src/main.rs

```diff
@@ -2980,16 +2980,19 @@ fn main() {
                     value_t_or_exit!(arg_matches, "snapshot_slot", Slot)
                 };
 
-                assert!(
-                    blockstore.meta(snapshot_slot).unwrap().is_some(),
-                    "snapshot slot doesn't exist"
-                );
+                if blockstore.meta(snapshot_slot).unwrap().is_none() {
+                    eprintln!(
+                        "Error: snapshot slot {} does not exist in blockstore or is not full.",
+                        snapshot_slot,
+                    );
+                    exit(1);
+                }
```

Comment:
The error message mentions "or is not full". This PR isn't checking that at the moment, but I think it would be a good idea to do so as that would also prevent us from being able to create a snapshot at some given slot

Reply:
hmm thought i did that, guess not. Done